modules/statistics/php/stats_demographic.class.inc

```diff
@@ -108,7 +108,7 @@ class Stats_Demographic extends \NDB_Form
         $tpl_data['TableHeader']   = $tableHeader;
         $tpl_data['Disclamer']     = $disclamer;
         $tpl_data['Subcategories'] = $subcats;
-        $tpl_var = \Utility::getSubprojectsForProject(intval($projectID));
+        $tpl_var = \Utility::getSubprojectsForProject(($projectID));
```

Comment:
Not a big issue but I think the extra parenthesis are not necessary here.

Reply:
Fixed